app/kumactl/cmd/install/testdata/install-control-plane.cni-experimental-enabled.golden.yaml

```diff
@@ -358,6 +358,149 @@ spec:
 ---
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.9.2
+  creationTimestamp: null
+  name: meshtraces.kuma.io
+spec:
+  group: kuma.io
+  names:
+    categories:
+    - kuma
+    kind: MeshTrace
+    listKind: MeshTraceList
+    plural: meshtraces
+    singular: meshtrace
+  scope: Namespaced
+  versions:
+  - name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: Spec is the specification of the Kuma MeshTrace resource.
+            properties:
+              default:
+                properties:
+                  backends:
```

Comment:
Again, I don't think this is a good idea and shouldn't be necessary. It can't be that we don't support a simple field like this in our policies. It's not a map, it's a sum type that our merging logic, if we can't just have custom logic per policy, must be able to support.

Reply:
@lobkovilya - I was only relying on your input here, can we support this?

Reply:
I tend to agree with Mike here. It's unfortunate that our technical limitations causes the api to be unintuitive.
An alternative could be to use backends and have "enabledBackends" which would work with merging.
Otherwise IIRC we said custom merging was one of these decisions we put out for later. Maybe this later is now...

Reply:
[note] decided on one element array during weekly for now, @lahabana to talk to @michaelbeaumont offline.